### DISCUSSION TITLE
Is there a  package to connect .NET Core 3.1 Server to a .NET 4.8 Clients?

### DISCUSSION THREAD

**Discussion by jcriddle-nci** [dotnet/aspnetcore#26618] [Q&A] [answered]
I am in search of a package that will allow a web application backend to communicate via TCP using .NET core 3.1 Server to talk to .NET 4.8 Clients.

I have seen that SignalR does not support communication between .NET and .NET Core.
WCF only works for .Net core clients.

Does anyone know of a package for communications between these two platforms?

**Comment by BrennanConroy** [comment, marked as answer]
>I have seen that SignalR does not support communication between .NET and .NET Core.

ASP.NET Core SignalR does not support talking to ASP.NET SignalR.

That doesn't mean you can't run the ASP.NET Core SignalR clients on .NET 4.8. The clients support netstandard2.0 (which is net461 I believe), and in 5.0 we added the net461 tfm to them directly.
